### PR TITLE
Permissions and actions updates

### PR DESCRIPTION
**Pull request by robertavram**

### CODE REVIEW

**Comment by robertavram** [comment]
@roman-karpovich if you could help with tests it would be awesome.
I'm relatively confident for now the permissions until the review state are working ok

File: src/etools/applications/partners/permissions.py [hunk 1]
```diff
@@ -118,6 +121,9 @@ def prp_mode_off():
         def prp_server_on():
             return tenant_switch_is_active("prp_server_on")
 
+        def unlocked(instance):
+            return not(instance.partner_accepted or instance.unicef_accepted)
```
Comment (reinbach) [inline src/etools/applications/partners/permissions.py:125]: Maybe move this logic to a `locked` method on the `Intervention` model, looks like we use this logic else where

File: src/etools/applications/partners/views/interventions_v3_actions.py [hunk 2]
```diff
@@ -248,19 +273,17 @@ class PMPInterventionUnlockView(PMPInterventionActionView):
     def update(self, request, *args, **kwargs):
         pd = self.get_object()
         request.data.clear()
+        if not(pd.partner_accepted or pd.unicef_accepted):
```
Comment (reinbach) [inline src/etools/applications/partners/views/interventions_v3_actions.py:276]: See note above about adding `locked` method on `Intervention` model

**Review by reinbach — APPROVED** [review]
Besides minor comment, and not going through the `intervention_permissions.csv` file too closely (makes me cross-eyed) looks good to me

**Review by roman-karpovich — APPROVED** [review]

**Pull request merged**